cpp/map_module/algorithm/map.cpp

```diff
@@ -0,0 +1,32 @@
+#include <list>
+#include <sstream>
+
+#include "map.hpp"
+
+void Map::FromPairs(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  mgp::memory = memory;
+  const auto arguments = mgp::List(args);
+  const auto record_factory = mgp::RecordFactory(result);
+  try {
+    const auto &list = arguments[0].ValueList();
```

Comment:
Remove reference here, and just use `const auto`, as `ValueList` returns temporary, we discussed that today on Slack. I think we will update that on Epic branch for others then

Reply:
https://memgraph.slack.com/archives/C05G7B5V1TR/p1690361110221839